lib/topological_inventory/ansible_tower/operations/core/authentication_retriever.rb

```diff
@@ -35,7 +35,9 @@ def process
               :headers => headers
             }
             response = RestClient::Request.new(request_options).execute
-            ::SourcesApiClient::Authentication.new(JSON.parse(response.body))
+            params = JSON.parse(response.body)
+            params.delete('tenant')
```

Comment:
because the public API doesn't return tenant (https://github.com/RedHatInsights/sources-api/pull/185) but internal API does. And the client is generated from public API